src/NexusMods.Icons/IconValue.cs

```diff
@@ -74,3 +82,5 @@ public record struct AvaloniaSvg(string? Path);
 [PublicAPI]
 public record struct AvaloniaPathIcon(Geometry? Geometry);
 
+[PublicAPI]
+public record struct SimpleVectorIcon(SimpleVectorIconImage? Image);
```

Comment:
Can be made non-null. The others are nullable because they can be used from XAML.

```suggestion
public record struct SimpleVectorIcon(SimpleVectorIconImage Image);
```